### ISSUE TITLE
Note when pressing tab in editor

### ISSUE THREAD

**Issue by amyjko** [wordplaydev/wordplay#758]
### What are you trying to do that you can't?

Some creators expect tab in the code editor to insert a tab, but tab is reserved for keyboard navigation of controls, per WCAG requirements. Because we can't and shouldn't remap tab to tab insertion, we need some way of explaining how to insert tabs (`ctrl-alt-tab`, the character insertion toolbar at the bottom).

### What is your idea?

When tab is pressed, show a note in the editor that explains the alternative ways to insert a tab, and hide that note on the next keystroke. We could also add an account-level setting that allows for tab to insert tabs, for those who aren't using keyboard support.

**Comment by yue7xu** [comment]
Hi @amyjko , can you assign this small design issue to me and @daminihs ? Here is our sketch to incoperate it into my current setting design in #730 :

![Image](https://github.com/user-attachments/assets/b4e55dc3-eaa6-43d9-912d-2f5e81c21ee8)

**Comment by amyjko** [comment]
(@daminihs will have to comment before I can add her).

I think a setting that permits this is a reasonable customization to allow creators who don't rely on the keyboard to navigate the interface to enable this efficiency.

**Comment by daminihs** [comment]
Hi @amyjko , I'd like to be assigned to this issue as well. Do we also want a small note/pop up to appear when 'Tab' is pressed while in the editor? We could design that interaction and share the sketch as well.

**Comment by yue7xu** [comment]
@amyjko I will continue to work on this during summer remotely.

**Comment by amyjko** [comment]
Sounds good, thanks @yue7xu. This should be a fairly straightforward design.

**Comment by yue7xu** [comment]
@amyjko Yes, thank you for checking in. I am sorry for the delay. I paused for a while and didn’t follow up, but I am picking it back up now. I will review the latest context and post a concrete update with next steps shortly.
